flavors/standard-EXASOL-7.1.0/flavor_base/testconfig

```diff
@@ -1,2 +1,2 @@
 generic_language_tests=python3 python java r
-test_folders=python python3/all python3/external_modules java r standard-flavor/all standard-flavor/7.1
+test_folders=python python3/all python3/external_modules java r standard-flavor/all standard-flavor/7.1 standard-flavor/7.0
```

Comment:
Hm, same thing, looks strange to add the test folder 7.0 to flavor standard-7.1.0